### PR TITLE
[MiniAppSDK][JS] Implement the interface miniAppFinishedLoading() to notify host app.

### PR DESCRIPTION
**Pull request by GondiTilak**
# Description
Implement the interface miniAppFinishedLoading() in the MiniApp JS SDK and js bridge.
This method should be used my the miniapps to notify the host app after it has finished loading.

# Checklist
- [x] I have completed all steps in the [Contribution Checklist](../blob/master.github/CONTRIBUTING.md#checklist)

### CODE REVIEW

File: js-miniapp-bridge/src/common-bridge.ts [hunk 1]
```diff
@@ -735,6 +735,22 @@ export class MiniAppBridge {
...
+  /**
+   * Associating miniAppFinishedLoading function to MiniAppBridge object.
+   * @returns Promise resolve with string
+   * Host app can implement an iterface miniAppFinishedLoading to perform any operations after the miniapp is loaded.
+   */
+  miniAppFinishedLoading() {
+    return new Promise<string>((resolve, reject) => {
+      return this.executor.exec(
+        'miniAppFinishedLoading',
+        {},
```
Comment (rleojoseph) [inline js-miniapp-bridge/src/common-bridge.ts:747]:
```suggestion
        null,
```

**Review by rleojoseph — CHANGES_REQUESTED** [review]
Looks ok to me. Only one small change. Also, please update ReadMe & Changelog for the same

**Comment by GondiTilak** [comment]
> Looks ok to me. Only one small change. Also, please update ReadMe & Changelog for the same

I have updated the change log and read me for the SDK.

File: js-miniapp-bridge/src/common-bridge.ts [hunk 2]
```diff
@@ -735,6 +735,22 @@ export class MiniAppBridge {
     });
   }
 
+  /**
+   * Associating miniAppFinishedLoading function to MiniAppBridge object.
+   * @returns Promise resolve with string
+   * Host app can implement an iterface miniAppFinishedLoading to perform any operations after the miniapp is loaded.
```
Comment (rleojoseph) [inline js-miniapp-bridge/src/common-bridge.ts:741]:
```suggestion
   * Host app can implement an interface miniAppFinishedLoading to perform any operations after the miniapp is loaded.
```

File: js-miniapp-sample/src/pages/landing.js [hunk 3]
```diff
@@ -101,6 +103,16 @@ const Landing = (props: LandingProps) => {
...
+  useEffect(() => {
+    try {
+      if (document.readyState === 'complete') {
+        miniAppDidFinishLoad();
+      }
+    } catch (e) {
+      console.log(e);
+    }
+  }, []);
+
```
Comment (rleojoseph) [inline js-miniapp-sample/src/pages/landing.js:115]: Try moving this to App.js

File: js-miniapp-sdk/README.md [hunk 4]
```diff
@@ -1186,6 +1186,32 @@ import MiniApp from 'js-miniapp-sdk';
...
 ```
 
+<div id='miniapp-finished-loading'/>
+
+## MiniApp Finished Loading <small style="color:green;font-size: 12px">Available from v1.20.0</small>
+
+<dl>
+<dd>
+
+**API:** [Platform.isDarkMode](api/interfaces/miniapputilsprovider.html#miniAppFinishedLoading)
```
Comment (rleojoseph) [inline js-miniapp-sdk/README.md:1196]: Please update it to use miniAppFinishedLoading

File: js-miniapp-sample/src/pages/landing.js (hunk 3 above)
Comment (GondiTilak) [reply js-miniapp-sample/src/pages/landing.js]: Done moved this to App.js and now showing the toast on success of this api call.

File: js-miniapp-sdk/README.md (hunk 4 above)
Comment (GondiTilak) [reply js-miniapp-sdk/README.md]: Updated thanks :)

File: js-miniapp-bridge/src/common-bridge.ts (hunk 2 above)
Comment (GondiTilak) [reply js-miniapp-bridge/src/common-bridge.ts]: updated ;)

**Review by rleojoseph — APPROVED** [review]

**Review by munir-rakuten — APPROVED** [review]

**Pull request merged**